### DISCUSSION TITLE
Board Firmware vs Display Firmware

### DISCUSSION THREAD

**Discussion by zbuh** [Jyers/Marlin#168] [Q&A] [answered]
I already compiled the firmware. 
But I dont find any information about the display firmware, shouldn't also be available a pre-compiled DWIN?

Am I missing something?

**Comment by Schnabulation** [comment, marked as answer]
For now, you don't need to update the displays firmware. This is only necessary when you want to change f.e. the icons. Jyers tried to not touch any icons so far so his firmware works perfectly fine with the stock display firmware.

**Reply by zbuh** [reply]
Ok. thanks.

I think it should be placed a note on the wiki stating that.

**Reply by Jyers** [reply]
Will do :)
